influxql/functions.go

```diff
@@ -936,27 +936,11 @@ type firstLastMapOutput struct {
 
 // MapFirst collects the values to pass to the reducer
```

Comment:
Let's add a comment here that states that `MapFirst` assumes the iterators are time-ordered.